cmd/run/flags.go

```diff
@@ -189,12 +189,19 @@ func bindRunFlagsFunc(flags *pflag.FlagSet) func(*cobra.Command, []string) {
 		util.MustBindPFlag("listUsersMaxResults", flags.Lookup("listUsers-max-results"))
 		util.MustBindEnv("listUsersMaxResults", "OPENFGA_LIST_USERS_MAX_RESULTS", "OPENFGA_LISTUSERSMAXRESULTS")
 
+		// TODO: make breaking change for cache limit
```

Comment:
To avoid breaking existing OpenFGA deployments with this release the cache size (which is now shared with subproblems and iterators) will still use its original flag. But the flag should be renamed in subsequent releases.